src/schema/firefox-schemas-import.js

```diff
@@ -565,42 +565,36 @@ function getTarballPath({ inputPath, version }) {
         .on('close', () => {
           resolve(tmpPath);
         });
-    } else {
-      reject(new Error('inputPath or version is required'));
-    }
-  });
+    });
+  }
+  throw new Error('inputPath or version is required');
```

Comment:
Nit, add an empty line between the closed bracket and the `throw new Error(...);`

Reply:
Done